doc/source/ray-core/fault_tolerance/gcs.rst

```diff
@@ -0,0 +1,69 @@
+.. _fault-tolerance-gcs:
+
+GCS Fault Tolerance
+===================
+
+Global Control Service (GCS) is a server that manages cluster-level metadata, such as locations of :ref:`actors <ray-remote-classes>`.
+It also provides a handful of cluster-level operations including :ref:`placement group <ray-placement-group-doc-ref>` scheduling and nodes management.
```

Comment:
I think we should also mention node management.

I categorize GCS functions into these:
- Node management
  - health check/register
  - resource broadcasting
- Scheduling/collaboration
  -  Actors/PGs
- Metastore for everything
- Others (might not need to mention)
  - Function manager
  - API server
  - ...